templates/adherent/home.html.twig

```diff
@@ -24,6 +24,14 @@
                 </p>
             </header>
 
+            <article class="l__wrapper--600 b__nudge--bottom-60 visually-hidden">
```

Comment:
1) this block is hidden with `visually-hidden`, normal?
2) I think you should move the if before `<article>` tag, if not you can have an empty `<article>` tag